src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  margin: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, that's the property you can change with * selector